jobs/JGLOBAL_ATMOS_VMINMON

```diff
@@ -17,16 +17,18 @@ export gcyc=${GDATE:8:2}
 #############################################
 # TANKverf - WHERE OUTPUT DATA WILL RESIDE
 #############################################
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_ANALYSIS
-YMD=${PDY} HH=${cyc} declare_from_tmpl -rx COM_ATMOS_MINMON
-YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx COM_ATMOS_MINMON_PREV:COM_ATMOS_MINMON_TMPL
+YMD=${PDY} HH=${cyc} declare_from_tmpl -rx \
+	COMIN_ATMOS_ANALYSIS:COM_ATMOS_ANALYSIS_TMPL \
+	COMOUT_ATMOS_MINMON:COM_ATMOS_MINMON_TMPL 
```

Comment:
```suggestion
	COMOUT_ATMOS_MINMON:COM_ATMOS_MINMON_TMPL
```

Reply:
@WalterKolczynski-NOAA  I need to make clear if the following modifications are correct:
1) original lines 20-21 were changed into new lines 20-21
2) original line 22 was changed into new lines 24-25
24 YMD=${gPDY} HH=${gcyc} declare_from_tmpl -rx \ 
25         COMIN_ATMOS_MINMON_PREV:COM_ATMOS_MINMON_TMPL

Reply:
The changes are correct. You just need to delete this trailing space.